src/graph.rs

```diff
@@ -350,12 +331,8 @@ fn hash_files(
         let mtime = file_state
             .get(id)
             .unwrap_or_else(|| panic!("no state for {:?}", graph.file(id).name));
-        let mtime_int = match mtime {
```

Comment:
I think we want to preserve the behavior where it crashes on missing mtime here, because we should never be calling this hasher in a scenario where a needed file is missing.